test/EmptyApp.FunctionalTests/project.json

```diff
@@ -1,31 +1,30 @@
 ﻿{
   "webroot": "wwwroot",
-  "version": "1.0.0-*",
+  "version": "1.0.0-rc2",
```

Comment:
BTW your own tests shouldn't need a version number. Can safely remove this from all your test projects unless they actually need to get published to a NuGet feed somewhere (which is generally unlikely).